cfme/tests/cloud/test_snapshot.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme import test_requirements
+from cfme.cloud.provider.openstack import OpenStackProvider
+
+
+@pytest.mark.manual
+@pytest.mark.provider([OpenStackProvider])
```

Comment:
This can be removed, as it is not paramterizing anything (this was a collect-only):
```
platform linux2 -- Python 2.7.15, pytest-3.4.1, py-1.7.0, pluggy-0.6.0 -- /home/juwatts/repos/cfme_venv/bin/python
cachedir: .pytest_cache
rootdir: /home/juwatts/repos/integration_tests, inifile:
plugins: polarion-collect-0.12, manageiq-integration-tests-17.43.1.dev411+g1aee60adb
collected 1 item                                                                                                                                                                                                                                                               
<Module 'cfme/tests/cloud/test_snapshot.py'>
  <Function 'test_osp_snapshot_buttons'>
Uncollection Stats:
 uncollectif: 0
 1 tests left after all uncollections

```

Reply:
/cloud/test_snapshot.py is a new file introduced by this PR. There will be more tests coming to this file later on. The test in this example should be run against OSP only. Are you sure we want to remove this?